src/logic/map_objects/tribes/productionsite.cc

```diff
@@ -421,15 +421,13 @@ InputQueue& ProductionSite::inputqueue(DescriptionIndex const wi, WareWorker con
 /**
  * Calculate statistic.
  */
-void ProductionSite::calc_statistics() {
-	// TODO(sirver): this method does too much: it calculates statistics for the
-	// last few cycles, but it also formats them as a string and persists them
-	// into a string for reuse when the class is asked for the statistics
-	// string. However this string should only then be constructed.
+void ProductionSite::format_statistics_string() {
+	// TODO(sirver): this method does too much: it formats the actual statistics
```

Comment:
There was an idea of "normalizing" percent like from range 3-97 to 0-100, this was not implemented, correct? If you are going to do this, this function would be the best place I think...

Reply:
I just wrote this morning that my tets have shown it reaches 0% but not 100% (mostly 98% although I have seen 99% as well. this is due to having an integer division in here which discards the remainder resulting in a floor function basically. so as we reach 0% by this I beleive we can live with 98% max productivity. But i will await feedback from the users on this.

Reply:
simple 'actual_statistics+5/10' would solve the rounding issue...

Reply:
I believe this would result in exactly the same as 5/10 is zero in integer division

Reply:
I dont uderstand, we would change
999/10 = 99
to 
(999+5) /10 = 100